evap/static/scss/components/_confirmation-modal.scss

```diff
@@ -11,6 +11,7 @@ dialog.evap-modal-dialog {
   border: 1px solid $dark-gray;
   border-radius: 0.5rem;
   padding: 0;
+  overflow: visible;
```

Comment:
Hm, do we want to do this on every confirmation modal? I guess most modals are well behaved and don't overflow anyways, so this should be fine; can you add a comment here so we know that the direct delegation modal needs this for its tomselect input?